scipy/integrate/_quad_vec.py

```diff
@@ -98,8 +98,9 @@ def __init__(self, **kwargs):
         self.__dict__.update(**kwargs)
 
     def __repr__(self):
-        return "_Bunch({})".format(", ".join(f"{k}={repr(self.__dict__[k])}"
-                                             for k in self.__keys))
+        return f"_Bunch({', '.join(
+            f'{k}={repr(self.__dict__[k])}' for k in self.__keys
+            )})"
```

Comment:
unfortunately you can't split this up over multiple lines in the middle of an f-string. Consider something like

```suggestion
	key_value_pairs = ', '.join(
            f'{k}={repr(self.__dict__[k])}' for k in self.__keys
        )
        return f"_Bunch({key_value_pairs})"
```